src/tests/specs/workItemTableTest.spec.ts

```diff
@@ -196,4 +196,21 @@ describe('Work Item datatable list: ', () => {
     expect(await planner.header.getFilterConditions()).toContain(labelFilter);
     expect(await planner.workItemList.datatableRow.count()).toEqual(countUnassignedWorkItem);
   });
+
+  it('Should filter the work item by close state', async () => {
+    let newWorkItem = {
+      title: 'Should filter the work item by close state - xxx'
+    };
+    let filterLabel = 'state: closed';
+    await planner.createWorkItem(newWorkItem);
+    expect(await planner.workItemList.hasWorkItem(newWorkItem.title)).toBeTruthy();
+    await planner.workItemList.clickWorkItem(newWorkItem.title);
+    await planner.quickPreview.changeStateTo('closed');
+    await planner.quickPreview.notificationToast.untilHidden();
```

Comment:
@vikram-raj could you please remove await planner.quickPreview.notificationToast.untilHidden();
Now only error notifications will be shown.
await planner.quickPreview.notificationToast.untilHidden();